packages/website/src/components/APISection.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {
+  Heading as RadixHeading,
+  Text,
+  Code,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@modulz/radix';
+
+export function APISection() {
+  return (
+    <>
+      <Heading>Props</Heading>
+      <Text size={3} sx={{ color: 'gray700' }}>
+        All components include the following props.
+      </Text>
+
+      <Table mb={4}>
+        <Thead>
+          <Tr>
+            <Th>Name</Th>
+            <Th>Type</Th>
+            <Th>Description</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          <Tr>
+            <Td>
+              <Code>sx</Code>
+            </Td>
+            <Td>Object</Td>
+            <Td>Theme-aware styles</Td>
+          </Tr>
+          <Tr>
+            <Td>
+              <Code>variant</Code>
```

Comment:
Does it make sense to have that one here? I imagine that's not something we want all DS to get.

Reply:
I dont follow, can you expand on what you mean?

Reply:
I mean that this API page has to work for all design systems right? I was anticipating this page to be generic so I found it surprising to see a row about a `variant` prop which only applies to radix. Of course it's fine here in radix, but ultimately this will be generated from Modulz, so just wondering if we'll be able to do that or not. Perhaps with the new styleguide editor we will?

Reply:
Yeah, feel you. I wouldnt be surprised if this page needed some work, but I just wanted to have _something_ there

Im currently working on the generated style guide, and will have to solve that there, but I think for Radix this is fine. 

What do you think?

Reply:
yep, all good 👍